ui/raidboss/data/06-ew/raid/p10s.ts

```diff
@@ -671,8 +671,8 @@ const triggerSet: TriggerSet<Data> = {
       'missingTranslations': true,
       'replaceSync': {
         'Arcane Sphere': 'sphère arcanique',
-        'Pand\\\\u00e6moniac Pillar': 'pilier pand\\u00e6moniaque',
```

Comment:
Thanks for the quick fix!

qq: If we already have `const bossNameUnicode = 'Pand\u00e6monium';` defined in the `ts` file, it might be better to just `[bossNameUnicode]: 'Pand\u00e6monium',` here to to ensure boss name is absolutely the same? I can submit a PR for this if you want. (I am sure this pr works, but just saying as a better code practice, we can even make this part of the RaidbossGuide translation guide to avoid issues like this in the future, for example "we recommend extract `replaceSync` keys as a shared variable to avoid mistype issues")?

Reply:
I think it is much more complicated than this, unfortunately. This is not a typo or a mistype, and the original entry is correct to translate timeline syncs. However, due to the way timelines vs triggers are parsed this same entry does not match the trigger. On top of that, there is an additional bug that find missing translations did not notice this mistake (when it normally should).

The way this bug would normally be detected is `missingTranslations` would not be set to true, and then `npm run test` would complain that there is not a match. However, one bug here hid the other.

It's also the case that due to translation collisions, the entry in `replaceSync` is not always the exact name of the boss. See  TOP for example, where `Omega` needs negative lookahead and lookbehind to not collide with `Omega-M` and `Alpha-Omega` and is not the literal key `Omega`. So, I don't think shared keys are generally the right solution.